frontend/app/src/utils/collection.ts

```diff
@@ -94,3 +98,15 @@ export const defaultOptions = <T extends Object>(
     sortDesc: [true]
   };
 };
+
+export const mapExchangeSavingsCollectionResponse = (
+  response: ExchangeSavingsCollectionResponse
+): ExchangeSavingsCollection => {
+  return {
+    found: response.entriesFound,
+    limit: response.entriesLimit,
+    total: response.entriesTotal,
+    data: response.entries,
+    ...response
+  };
+};
```

Comment:
Any reason we can't merge this with the other function above?
It should be possible to just use a single function for all cases

Reply:
btw if you merge them maybe do a `const { entriesFound, entriesLimit, entriesTotal, entries, ...rest } = x`

and then return `{ found: entriesFound, ..., ...rest }`

because this was the response there would still pass all the properties of the original.